docs/upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -40,11 +40,13 @@ There you can find links to upgrade notes for other versions too.
         ```
 
 ### Database migrations
-- after running database migrations, products will have new columns for "Selling start date" (selling_from_date_time) and "Selling end date" (selling_to_date_time) as DateTime fields ([#1343](https://github.com/shopsys/shopsys/pull/1343))
+- after running database migrations, products will use a DateTime type for columns for "Selling start date" (selling_from) and "Selling end date" (selling_to) ([#1343](https://github.com/shopsys/shopsys/pull/1343))
```

Comment:
UPGRADE notes should only include instructions what users are supposed to do

Reply:
fixed in b5bff4734bd5938a3e57b9ccc7f8111778a21fa5